BestPractices.md

```diff
@@ -213,4 +214,10 @@ When you document some code that is coming from a tutorial or is referenced in a
 
 #### Annotations ####
 
-You can provide different annotations to inform other developers about the code they are looking at. Different keywords are available, such as `important` to highlight critical information, `note` to provide additional information, `warning` to warn other developers about this piece of code.
\ No newline at end of file
+You can provide different annotations to inform other developers about the code they are looking at. Different keywords are available, such as `important` to highlight critical information, `note` to provide additional information, `warning` to warn other developers about this piece of code.
+
+### Access Control ###
+
+When choosing access levels for types defined within your project, follow Apple's [recommendations](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/AccessControl.html). For single-target apps, `internal` or a more restrictive access level should be used. For frameworks and code that need to be accessed from a different module, expose the public-facing interface using `public` or `open`. Implementation details such framework can and should still be hidden with the default access level of `internal`.
+
+Start with the most restrictive access level, `private`, and increase access as needed (`private` -> `fileprivate` -> `internal`), when defining member constants, variables or functions. If a type's member or function needs to be accessed from a protocol conformance extension within the same file as described in [this section](https://github.com/prolificinteractive/swift-style-guide#file-structure), use the access level of `fileprivate`.
```

Comment:
it would be interesting to document all the access control keywords from the most private to the most public, basically continue what you started here in its own sub-section